documentation/meta/documentation/adding_docs.md

```diff
@@ -0,0 +1,34 @@
+# Adding a new documentation page
+
+This is a quick guide for adding a new documentation page and running the
+documentation locally.
+
+## Prerequisites
+
+Refer to the [general setup guide](/general/general_setup.md) for setting up the
+prerequisites. Refer to the 'Docs' column in the
+[requirement matrix](/general/general_setup.md#requirement-matrix) to know what
+you need to run this.
+
+## Starting up
+
+The following demonstrates the process of adding a new documentation page.
+
+1. Create a markdown file at a place within the `documentation/` hierarchy that
+   makes the most sense for your case, this would usually already contain a
+   `index.md` file, make sure that your new file is located next to it.
```

Comment:
Wording suggestion:
```suggestion
   makes the most sense for your case. this would usually already contain a
   makes the most sense for your case. Usually the parent folder for the new
   file would usually already contain an `index.md` file, make sure that your
   new file is located next to it. If a new directory is required, you will
   also need to create an `index.md` file for it with a table of contents for
   the folder. See other `index.md` files within the `documentation/`
   directory for examples.
```